public/css/app.css

```diff
@@ -1,268 +1,17 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto);
-
-/*!
- * Bootstrap v4.0.0-alpha.6 (https://getbootstrap.com)
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500);/*!
```

Comment:
This file should not be commited. At the deployment the assets will be generated.

Reply:
Remove also the public/js/app.js